src/layouts/product-integration-layout/components/header/index.tsx

```diff
@@ -49,41 +59,83 @@ export default function Header({
 		.map((f: Flag) => f.slug)
 		.includes('builtin')
 
+	const tags = GetIntegrationTags(integration, true)
+
 	return (
-		<Card className={classNames(s.header, className)}>
-			<div className={s.upperCard}>
-				<div className={s.left}>
-					<h1>{integration.name}</h1>
-					<span>@{integration.organization.slug}</span>
-				</div>
-				<div className={s.right}>
-					{showVersions ? (
-						<DropdownDisclosure
-							className={s.versionDropdown}
-							color="secondary"
-							text={versionString(activeRelease.version, integration.versions)}
+		<Card elevation="base" className={s.headerWrap}>
+			<IconTileLogo
+				className={s.iconTile}
+				size="large"
+				productSlug={productSlug !== 'sentinel' ? productSlug : null}
+			/>
+			<div className={s.upper}>
+				{showVersions ? (
+					<DropdownDisclosure
+						className={s.versionSelector}
+						color="secondary"
+						text={versionString(activeRelease.version, integration.versions)}
+					>
+						{otherVersions.map((version: string) => {
+							return (
+								<DropdownDisclosureLinkItem
+									key={version}
+									href={getVersionChangedURL(version)}
+								>
+									{versionString(version, integration.versions)}
+								</DropdownDisclosureLinkItem>
+							)
+						})}
+					</DropdownDisclosure>
+				) : null}
+				<div className={s.titleDescriptionWrap}>
+					<div className={s.titleWrap}>
+						<TruncateMaxLines
```

Comment:
The designs were very specific about truncation behavior, and they differ betwen mobile / desktop views for both the title and description.